### PR TITLE
fix:other acc transfer to own acc

### LINKED ISSUE #713: UI Issue while adding Transfer
**Summary:**

UI Issue while adding Transfer

**Steps to reproduce:**

1) Click on Transfer
2) Choose Transfer from Transfer Type

**Expected behaviour:**

End user should not be able to add money to his/her account from other users accounts due to UI Inconsistency

**Observed behaviour:**

End user can add money to his/her amount from other users accounts due to UI Inconsistency

**Device and Android version:**

Xiaomi, Marshmallow

**Screenshots:**

_issue_security_1

_issue_security_2

### PR DESCRIPTION
**Pull request by PiyushSharma99**
Fixes #713 

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by PiyushSharma99** [comment]
@jddeep @santoshconflux @GOVINDDIXIT please review and approve. Thanks!

**Review by jddeep — APPROVED** [review]
Looks good 👍 Please format the code correctly.

**Comment by laxyapahuja** [comment]
Can you upload a screen recording that shows that you've fixed the error?

File: app/src/main/java/org/mifos/mobile/ui/fragments/SavingsMakeTransferFragment.java [hunk 1]
```diff
@@ -175,7 +175,7 @@ public void onCreate(Bundle savedInstanceState) {
 
     @Override
     public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
-            @Nullable Bundle savedInstanceState) {
+                             @Nullable Bundle savedInstanceState) {
```
Comment (ShivangiSingh17) [inline app/src/main/java/org/mifos/mobile/ui/fragments/SavingsMakeTransferFragment.java:178]: Remove the unnecessary spacing.

File: app/src/main/java/org/mifos/mobile/ui/fragments/SavingsMakeTransferFragment.java [hunk 2]
```diff
@@ -397,6 +397,12 @@ public void onNothingSelected(AdapterView<?> parent) {
      */
     @OnClick(R.id.btn_pay_to)
     public void payToSelected() {
+        String payToIdString = payTo.replaceFirst("^0+(?!$)", "");
```
Comment (ShivangiSingh17) [inline app/src/main/java/org/mifos/mobile/ui/fragments/SavingsMakeTransferFragment.java:400]: Why did you add this?
